src/6model/6model.h

```diff
@@ -134,7 +134,10 @@ typedef enum {
     /* Note: if you're hunting for a flag, some day in the future when we
      * have used them all, this one is easy enough to eliminate by having the
      * tiny number of objects marked this way in a remembered set. */
-    MVM_CF_NEVER_REPOSSESS = 2048
+    MVM_CF_NEVER_REPOSSESS = 2048,
+
+    /* Does this use the FSA? */
+    MVM_CF_USES_FSA = 4096
```

Comment:
I'm working on a branch where I use the flag to mean the same thing for `MVMStrings` (their body.storage is FSA allocated). I could rename it here and `#define` it in `MVMString.c` also, but it seems like it would also make sense as a flag for any `MVMCollectable`?

Reply:
`#define` it for `MVMString.c` also. Two REPRs out of a couple of dozen isn't really persuasive. :-) I can think of something else I'd like to use such a flag for specific to `P6opaque` in the not too distant future, so could use that bit for it. We've used most of the bit field already, so I'd rather keep them general.

Reply:
Sure, easy enough. I'll update the PR this evening.